src/modules/search/components/Search.jsx

```diff
@@ -30,26 +30,13 @@ class Search extends React.Component {
 const SearchHolder = styled.div`
 	background: rgba(36, 36, 36);
 	box-shadow: 0px 4px 6px 4px rgba(0, 0, 0, 0.1);
-	display: block;
-	position: absolute;
-	top: 12.125rem;
-	width: 100%;
-	@media ${device.small} {
-		top: 4.675rem;
-	}
 `;
 
 const SearchBackground = styled.div`
 	background: rgba(25, 25, 25, 0.97);
-	bottom: 0;
-	content: '';
-	left: 0;
-	position: fixed;
-	top: 0;
 	width: 100%;
-	@media ${device.small} {
-		top: 4.675rem;
-	}
+	height: 100vh;
```

Comment:
the problem with the search background only being height: 100vh; is that if scroll the page when search is open, it looks like this 

<img width="1297" alt="screenshot 2018-10-09 23 35 49" src="https://user-images.githubusercontent.com/7230519/46711864-3d09c080-cc1c-11e8-8d2e-2b16ad840237.png">

We can prevent scrolling of the body as a way to get around this, but it was the reason i did position fixed.